ta/pkcs11/src/pkcs11_token.h

```diff
@@ -142,6 +142,23 @@ struct active_processing {
 	void *extra_ctx;
 };
 
+/*
+ * Pkcs11 objects search context
+ *
+ * @attributes - matching attributes list searched (null if no search)
+ * @count - number of matching handle found
+ * @handles - array of handle of matching objects (published handles)
+ * @next - index of the next object handle to return to FindObject
+ * @temp_start - index of the trailing not yet published handles
```

Comment:
maybe rename it, maybe s/temp_start/no_hdl_start/.

Reply:
Removed temp_start all together